lib/process.py

```diff
@@ -68,6 +68,12 @@
 parser.add_argument("-prefix", help="Prefix to add to the generated PSyData "
                                     "function names",
                     default="")
+parser.add_argument("-generic-declare", action="store_true",
```

Comment:
Please could you add comments explaining why these arguments are needed.

Reply:
Done.